clearwater-config-manager.root/usr/share/clearwater/clearwater-config-manager/scripts/is_config_synchronized

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
I don't understand this script. Is this because check_config_sync always returns success and you want the return code? In which case can you change check_config_sync to have a useful return code instead?